app/styles/courses/teacher-class-view.sass

```diff
@@ -0,0 +1,250 @@
+@import "app/styles/bootstrap/variables"
+@import "app/styles/mixins"
+@import "app/styles/style-flat"
+
+$grey-light: #f5f5f5
+$grey-medium: #ebebeb
+$grey-dark: #c9c9c9
+$grey-darker: #9b9b9b
+
+.alternating-background:nth-child(2n+1)
+  background-color: $grey-medium
+
+.alternating-background:nth-child(2n)
+  background-color: $grey-light
+
+#teacher-class-view
+
+  ul.nav-tabs
+    border: none
+    .tab-spacer
+      float: left
+      width: 15px
+      height: 47px
+      border-bottom: thin solid #979797
+      
+    .tab-filler
+      // Triggers block formatting context
+      // http://stackoverflow.com/questions/1260122/expand-a-div-to-take-the-remaining-width
+      overflow: hidden
+      height: 47px
+      border-bottom: thin solid #979797
+      
+    li > a
+      margin: 0
+      border: thin solid #979797
+      border-bottom: none
+      background: white
+      border-radius: 5px 5px 0 0
+      width: 175px
+      color: black
+      font-family: $headline-font
+    li
+      border-bottom: thin solid #979797
+    li.active
+      border-bottom: none
+  
+  .bulk-assign-controls
+    float: right
+    margin-bottom: -9999px
+    margin-top: 20px
+    select
+      margin-left: 7px
+    .assign-to-selected-students
+      margin-left: 10px
+    .enroll-selected-students
+      margin-left: 56px
```

Comment:
I just merged into master my latest global changes from teacher-accounts, which includes the spacing classes. Going forward at least, use them instead of these sorts of one-off rules.